adapters/jwplayer/contentTargeting.go

```diff
@@ -41,7 +41,7 @@ type TargetingData struct {
 	TargetingProfiles []string `json:"targeting_profiles"`
 }
 
-type Enricher interface {
+type RTDAdapter interface {
```

Comment:
does this interface belong in this file?

Reply:
ContentTargeting and jwplayer are all part of the same package, so the difference is only lexical. I could see it going in its own file, or in jwplayer.go